users/create.go

```diff
@@ -158,6 +166,29 @@ func createUserWithValidBody(request *http.Request, body createUserRequestBody,
 	user, err := CreateUserWithPassword(ctx, tx, body.Email, body.Password, customerIdentifier)
 	if err == nil {
 		logger.Info("User created.", user)
+		entitlement.CheckUserEntitlements(request.Context(), tx, user.Id)
+		return 200, user
+	} else {
+		logger.Error(err.Error(), nil)
+		errSplit := strings.Split(err.Error(), ":")
+		if len(errSplit) >= 1 && errSplit[0] == "Error 1062" {
+			return 409, errors.New("Account already exists.")
```

Comment:
Is it the right HTTP code? Is it the one we use elsewhere?

Reply:
Yes, it was already there in the ```createUserWithValidBody``` function.